eng/PR-15 Show total count and redirect to trade alerts if non-CO

gcruicks opened the pull request:
Route a smaller message and send a user to trade alerts if a non carrier owner uses the command m.issions
Fixes: #15

cimspin reviewed (changes requested):
- Seems to be listing the loading missions (but not unloading missions) when used by a non CO.
- Also can we change the text for non-COs to "Please follow the link to <trade-alerts-channel-link> for full details of all current missions."
Looks great though!

gcruicks commented:
> * Seems to be listing the loading missions (but not unloading missions) when used by a non CO.
> * Also can we change the text for non-COs to "Please follow the link to  for full details of all current missions."
>   Looks great though!

Changed the message for non-CO and added a different string if no ongoing messages at all.
It displays the total count of missions, so it covers both scenarios I think today, where did you see it only covering the loading?
